main.cpp

```diff
@@ -1,162 +1,241 @@
+    see main() for an example of this.
+ 
+10) click the [run] button.  Clear up any errors or warnings as best you can.
+ */
+
+//2)
+void variableDeclarations()
+{
+    //example:
+    int number = 2; //declaration of a variable named "number", that uses the primitive type 'int', and the variable's initial value is '2'
+    int value = 1;
+    int param = 9;
+
+    float size = 0.3f;
+    float volume = 0.22222f;
+    float length = 0.27981f;
+
+    bool toggleSwitch = 0;
+    bool pressButton = 1;
+    bool touchSense = 1;
+
+    double preciseValue = 0.445555;
+    double accurateData = 0.3432552;
+    double serialData = 0.00235252;
+
+    unsigned int positiveValue = 3;
+    unsigned int positiveNum = 2;
+    unsigned int count = 1;
+
+
+    ignoreUnused(number, value, param, size, volume, length, toggleSwitch, pressButton, touchSense, preciseValue, accurateData, serialData, positiveValue,positiveNum, count); //passing each variable declared to the ignoreUnused() function
+}
+
+/*
+ 10 functions
+ example:
+ note: this example shows the result after completing steps 3-8
+ */
+
+bool rentACar(int rentalDuration = 0, int carType = 0)  //function declaration with random number of arguments, arbitrary number of arguments have default value
+{ 
+    ignoreUnused(rentalDuration, carType); //passing each function parameter to the ignoreUnused() function
+    return {}; //if your function returns something other than void, add 'return {};' at the end of it.
+} 
+
+/*
+ 1)
+ */
+void soundOsc (int waveType = 1, double amplitude = 0.1) 
+{
+    ignoreUnused(waveType, amplitude);
+
+}
+/*
+ 2)
+ */
+double sumLR (double leftCh = 0, double rightCh = 0)
+{
+    ignoreUnused(leftCh, rightCh);
+    return{};
+}
+
+/*
+ 3)
+ */
+int waveType (int waveSelSw = 0) 
+{
+    ignoreUnused(waveSelSw);
+    return{};
+}
+
+/*
+ 4)
+ */
+double amplitudeEnv (double ampEnv = 0.1, double oscMixerOut = 0)
+{
+    ignoreUnused(ampEnv, oscMixerOut);
+    return{};
+}
+
+/*
+ 5)
+ */
+bool selectParam (bool switchOne = 0, bool switchTwo = 0)
+{
+    ignoreUnused(switchOne, switchTwo);
+    return{};
+} 
+
+/*
+ 6)
+ */
+float displaySelectedOscType (bool switchOne = 0, int encoderSel = 0)
+{
+    ignoreUnused(switchOne, encoderSel);
+    return{};
+}
+
+/*
+ 7)
+ */
+bool isSomethingConnected (bool inputOneSwitch = 0, bool inputTwoSwitch = 0)
+{
+    ignoreUnused(inputOneSwitch, inputTwoSwitch);
+    return{};
+}
+
+/*
+ 8)
+ */
+int checkBatteryStatus (int batteryTemp = 0, int batteryCharge = 0, bool batteryCharging = 0)
+{
+    ignoreUnused(batteryTemp, batteryCharge, batteryCharging);
+    return{};
+}
+
+/*
+ 9)
+ */
+int detectInputType (float inputImpedance = 0, double inputAmplitue = 0.1)
+{
+    ignoreUnused(inputImpedance, inputAmplitue);
+    return {};
+}
+
+/*
+ 10)
+ */
+int activateArp (bool isMidiInconnected = 0, int arpSelType = 0)
+{
+    ignoreUnused(isMidiInconnected, arpSelType);
```

Comment:
Booleans need to be initialized with `true` or `false`. 
never initialize booleans with integers
- [ ] revise